CHANGELOG.md

```diff
@@ -12,9 +12,8 @@ x.x.x
 - Add support for attaching existing FSx for Ontap and FSx for OpenZFS File Systems.
 - Add support for FSx Lustre Persistent_2 deployment type.
 - Show `requested_value` and `current_value` values in the change set when adding or removing a section.
-- Add new configuration parameter `Scheduling/SlurmSettings/EnableMemoryBasedScheduling` to configure memory-based
-  scheduling in Slurm.
-  - Add new configuration parameter to override default value of schedulable memory on compute nodes.
+- Add new configuration parameter `Scheduling/SlurmSettings/EnableMemoryBasedScheduling` to configure memory-based scheduling in Slurm.
```

Comment:
These changes in the config parameters are more important than requested_value and current_value change.


Then the requested_value and current_value is more a change than an enhancement in my opinion. It has very low impact for the customer.